ui/apps/platform/src/Components/CompoundSearchFilter/components/SearchFilterAutocomplete.tsx

```diff
@@ -144,7 +144,7 @@ function SearchFilterAutocomplete({
     );
     // Filter out empty strings
     const data: SearchAutocompleteQueryResponse = {
-        searchAutocomplete: rawData?.searchAutocomplete?.filter((item) => item !== '') ?? [],
+        searchAutocomplete: rawData?.searchAutocomplete?.filter((item) => item !== '').sort() ?? [],
```

Comment:
Remind me, optional chaining stops at first `undefined` value, correct?
Therefore, needed for `?.filter(…)` but not needed for `.sort(…)`

Reply:
Correct 👍🏼